vulnerabilities/data_dump.py

```diff
@@ -68,3 +70,55 @@ def ubuntu_dump(html):
             vulnerability=vulnerability,
             package=package
         )
+
+
+def archlinux_dump(extract_data):
+    """
+    Save data scraped from archlinux' security tracker.
+    Args:
+        extract_data(generator): data collected from archlinux' security tracker
+    """
+    for data in extract_data:
+        packages_name = data['packages_name']
+        vulnerabilities = data['vulnerability_id']
+        affected_version = data['version'][0]
+        fixed_version = data['version'][1]
+
+        vulnerability = Vulnerability.objects.create(
+            summary=data['description'],
+        )
+
+        for vulnerability_id in vulnerabilities:
+            VulnerabilityReference.objects.create(
+                vulnerability=vulnerability,
+                reference_id=vulnerability_id,
+                source='archlinux',
+            )
+
+        for package_name in packages_name:
+            package_affected = Package.objects.create(
+                name=package_name,
+                version=affected_version
+            )
+            ImpactedPackage.objects.create(
+                vulnerability=vulnerability,
+                package=package_affected
+            )
+            PackageReference.objects.create(
+                package=package_affected,
+                repository='https://security.archlinux.org/package/'+
+                            '%s'% package_name
+            )
+            package_fixed = Package.objects.create(
+                name=package_name,
+                version=fixed_version
+            )
+            ResolvedPackage.objects.create(
+                vulnerability=vulnerability,
+                package=package_fixed
+            )
+            PackageReference.objects.create(
+                package=package_fixed,
+                repository='https://security.archlinux.org/package/'+
+                            '%s'% package_name
```

Comment:
BTW, why
 ```
'https://security.archlinux.org/package/'+
                            '%s'% package_name
```
and not
`'https://security.archlinux.org/package/%s' % package_name`
both here and above?